addons/medical_feedback/functions/fnc_effectBloodVolumeIcon.sqf

```diff
@@ -0,0 +1,44 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles the blood volume icon.
+ *
+ * Arguments:
+ * 0: Enable <BOOL>
+ * 1: Intensity 0...6 <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [true, 4] call ace_medical_feedback_fnc_effectBloodVolumeIcon
+ *
+ * Public: No
+ */
+
+params ["_enable", "_intensity"];
+
+private _indicatorCtrl = uiNamespace getVariable [QGVAR(bloodVolumeIndicator), controlNull];
+if (isNull _indicatorCtrl) exitWith {};
+
+if (!_enable || !GVAR(showBloodVolumeIcon)) exitWith {
+    if (ctrlText _indicatorCtrl != "") then { 
+        _indicatorCtrl ctrlSetText "";
+    };
```

Comment:
Why do you check if text != "" when you set it to "" anyway? Seems like the check is not needed.

Reply:
If != "" then indicator is displayed and i remove it, if it's empty then no extra ctrlSetText calls. Ofc it's minor thing, but why not?

Reply:
It's unnecessary, think:
- If text is empty, check is `false`, so it stays empty.
- If text is NOT empty, check is `true` and it changes to empty.

Both result in empty text, so only setting it to empty without the check gives the same result.

Reply:
Is there performance hit in this though?
we already know some ctrl commands are stupidly expensive for no discernable reason.

Reply:
Removed